authn/const.go

```diff
@@ -6,4 +6,6 @@ const (
 
 	httpHeaderAccessToken = "X-Access-Token"
 	httpHeaderIDToken     = "X-Grafana-Id"
+
+	ServiceIdentityKey = "serviceIdentity"
```

Comment:
Nit. Should we be a bit more precise?
```suggestion
	ServiceNameExtraKey = "serviceIdentity"
```